website/docs/advanced-concepts/access-control/granular-access-control/roles.md

```diff
@@ -0,0 +1,242 @@
+# Roles
+ A Role is a collection of permissions that grant access to perform certain operations on resources and can be assigned to a user or a group. Roles enable users to manage multiple permissions rather than assigning individual permissions to users/groups. In Appsmith, a role can be mapped to multiple permissions and a permission can be mapped to multiple roles. However, a role cannot be nested, i.e. it can contain only permissions and not other roles. Roles are useful for efficiently managing permissions to access Appsmith resources.
+ 
+ Appsmith provides three built-in roles - **Admin**, **Developer** and **App Viewer**.  With Granular Access Contorl (GAC), you can create a custom role that provides fine-grained access control. 
```

Comment:
Are we intentionally keeping this **Admin** and not **Administrator**?

Reply:
Sorry, I missed it. Fixing it now